client/src/components/FoodCategory/index.js

```diff
@@ -0,0 +1,59 @@
+import Chip from "@mui/material/Chip";
+import Fade from "@mui/material/Fade";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Switch from "@mui/material/Switch";
+
+export default function FoodCategories(foodCategories) {
+  const [checked, setChecked] = React.useState(false);
+
+  const DEFAULT_FOODS = {
+    Halal: false,
+    Chinese: false,
+    Mexican: false,
+    Amongus: false,
+    Imposter: false,
+  };
+
+  const [foodCategory, setFoodCategory] = React.useState(DEFAULT_FOODS);
+  const foods = Object.keys(foodCategory);
+  const selectedFoods = Object.keys(foodCategory).filter(
+    (food) => foodCategory[food]
+  );
```

Comment:
Thought I'd need this somewhere but don't think so

Reply:
We'll need it later